### ISSUE TITLE
definition of present and past weather varaiables

### ISSUE THREAD

**Issue by meulenvd** [wmo-im/wmds#364]
_Please consult the_ [_Guidance_](https://github.com/wmo-im/wmds/wiki/Changing-Codetables) _and use the following structure for new issues. Note, the content in this initial comment box is subject to modification to document the final decisions or to provide more details as needed._

# **Branch**

\[add branch when created\]

# **Summary and Purpose**

\[add brief overview\]

# **Stakeholder(s)**

\[include emails or handles as relevant\]

# **Proposal**

to implement definition for the specific weather phenomena and present weather variables, as defined in WMO-No. 488,  
 the Int. Cloud Atlas and WMO-No. 8 following WMO-No. 1160. Considering UDC as well.

# **Reason**

Np definitions available yet

**Comment by amilan17** [comment]
https://library.wmo.int/doc_num.php?explnum_id=3160

**Comment by joergklausen** [comment]
@fstuerzl Please create branch
@meulenvd Please review branch
@gaochen-larc, @ferrighi  Please confirm

**Comment by fstuerzl** [comment]
New definitions added in branch: https://github.com/wmo-im/wmds/commit/f9233cf2d006bdbe095ca4dad30a22bc97f5aafa

**Comment by gaochen-larc** [comment]
I confirm that the proposed names and descriptions are appropriate.

**Comment by joergklausen** [comment]
I confirm this branch is correct.

**Comment by joergklausen** [comment]
@meulenvd Your confirmation would be valuable.

**Comment by amilan17** [comment]
I think it will be better to use the WMO-No. 182 definitions. 

Also, I think it's a best practice to NOT include URLs in the definitions.

**Comment by joergklausen** [comment]
@meulenvd To me, No. 8 is equally normative, but if you prefer to refer to WMO-No. 306, Vol. I.1, I don't have a problem with that. I do have a problem with the reference to SYNOP in the definition in WMO. 182, though, because we cannot define a term with reference to another term that is not defined. Please update the branch and the first comment in the issue, then ask us for confirmation. We should conclude this ASAP.

**Comment by meulenvd** [comment]
Yes the reference to SYNOP in WMO-No. 182 is a bit strange, but understandable. This is because reporting weather phenomena where requested already during the IMO area and therefore is present  and past weather always associated to data exchange (and therefore defined in the Manual on Codes).

Other examples I found are:

WMO-No. 544 Manual on the GOS (obsolete; definition modified when transferring it into the Manual on the WIGOS)
_Weather:_ State of the atmosphere at a particular time, as defined by the various meteorological
elements.
_– Present weather:_ Weather existing at a station at a time of observation.
_– Past weather:_ Predominant characteristic of the weather which existed at an observing station during a given period of time.

WMO-No, 1160 (2021) Manual on the WIGOS:
_Present and past weather_: The qualitative description of observable phenomena, at the time of observation or during a preceding period. 
**Note:** Relevant observable phenomena in the atmosphere include precipitation, suspended or blowing particles, and other designated optical phenomena or electrical manifestations, as described in the International Cloud Atlas – Manual on the Observation of Clouds and other Meteors (WMO-No. 407), the Guide to Instruments and Methods of Observation (WMO-No. 8) and, for aeronautical applications, in the Technical Regulations (WMO-No. 49), Volume II.

So I think the definition given in the Manual on the WIGOS is the most actual version (present and past weather given by one definition) and has preference. In fact it combines the defition of the ols Man. GOS into one.

**Comment by amilan17** [comment]
I vote for the definitions in MoC

\[2\] WMO-No. 306, Vol. I.1 Manual on Codes, PART A – Alphanumeric Codes  
Present weather: Weather existing at the time of observation, or under certain conditions, during the hour preceding the time of observation.  
Past weather: Predominant characteristic of weather which had existed at the station during a given period of time

**Comment by joergklausen** [comment]
In support of @amilan17 .

**Comment by ferrighi** [comment]
WMO-No. 306 seems the best set of definitions.

**Comment by fstuerzl** [comment]
@amilan17, I've included the definitions from MoC in the proposal above.

**Comment by gaochen-larc** [comment]
> In support of @amilan17 .

I second

**Comment by meulenvd** [comment]
The best choice for Present and Past Weather is a bit a dillemma for me. The definition in the WMO-No. 360, Manual on Codes (Vol I) is exactly the same as in WMO-No. 544 Manual on the GOS (now obsolete). When transferring the Manual on the GOS into WMO-no. 1160, the Manual in the WIGOS it was decided by the EdBd to update the definition into a more generic form (see my comment above; **Present and past weather: The _qualitative_ description of observable phenomena, at the time of observation or during a preceding period**.). 
The term Present and Past Weather has a long history (> 70 a) and everything to do with reporting (e.g. FM12 - Synop). Because of it limitations (short bulletins send over telex or telegraph) and reporing frequency (e.g. only at the main hours) it was necesary to report a simple code (well known as de WW code). This code should contain the most significant weather at the station during the observation interval, but because of the long reporting interval also significant weather (e.g.a thunderstorm 2 hours ago) should be reported (as past weather). For each WW (00-99) there is a specific symbol designed, to be used in the synoptic weather maps. Relevant is that present & past weather belong to the subset of "subjective observations". These are observations that cannot be reported as "objective", quantitative variables like e.g. temperature (therefore: _qualitative_ descriptions). Note that today most of the items in WW (or WaWa for AWS) can be reported quantitatively (precipitation intensity, lightning, obstruction to vision) and it is recommended to do so _only_. This is the major reason why the definition was modified when transferring the Man GOS into Man WIGOS.

**Comment by amilan17** [comment]
If it helps, a quick search in OSCAR/Surface results in 8448 stations / platforms found with these codes.

**Comment by amilan17** [comment]
[https://github.com/wmo-im/wmds/wiki/2022-05-19-TT-WIGOSMD-22](https://github.com/wmo-im/wmds/wiki/2022-05-19-TT-WIGOSMD-22) notes: 

*   use the definitions from MoC vI.1
*   list all publications consulted in the summary

**Comment by joergklausen** [comment]
Proposal approved by team, summary will be amended to reflect sources considered.

**Comment by fstuerzl** [comment]
I've included the context in the proposal summary, citing the various definitions and sources that were mentioned.

**Comment by amilan17** [comment]
@joergklausen - ready to validate

**Comment by amilan17** [comment]
[https://github.com/wmo-im/wmds/wiki/2022-05-31-TT-WIGOSMD-23](https://github.com/wmo-im/wmds/wiki/2022-05-31-TT-WIGOSMD-23) notes: @joergklausen validate

**Comment by amilan17** [comment]
@meulenvd -- this link (https://library.wmo.int/doc_num.php?explnum_id=3160) takes the user to a copy of a chapter of a publication. Do you know which WMO Publication it came from?

**Comment by joergklausen** [comment]
> WMO-No. 544 Manual on the GOS

https://library.wmo.int/doc_num.php?explnum_id=10616

**Comment by meulenvd** [comment]
@amilan17 - the link (https://library.wmo.int/doc_num.php?explnum_id=3160) refers to _Chapter 14_ of **WMO-No. 8, Vol I.** (GIMO, WMO Guide to Instruments and Methods of Observation). A better is link to WMO-No. 8 (2018 ed.) is (https://library.wmo.int/index.php?lvl=notice_display&id=12407) for all 5 Volumes, or (https://library.wmo.int/doc_num.php?explnum_id=10616) for the download of the Vol I pdf. 
( @joergklausen  - for info: the Man on the GOS (WMO-No. 544) is obsolete but largerly merged into the Man on the WIGOS (WMO-No. 1160))
